commitments/eigenda.go

```diff
@@ -21,7 +21,7 @@ type CertCommitment interface {
 type EigenDACommitment struct {
 	prefix   EigenDACommitmentType
 	b        []byte
-	encoding *EncodingType // Optional encoding type, used only for V2+
+	encoding EncodingType // Optional encoding type, used only for V2+
```

Comment:
```suggestion
	encoding EncodingType // encoding type, used only for V2+
```

Reply:
not optional anymore right?

Reply:
also is it really only used for v2? doesnt ssem for v2 only in meta struct
> 	// encoding type for the certificate (defaults to RLPEncoding)
	Encoding EncodingType